test/geometries/linestring.cpp

```diff
@@ -0,0 +1,95 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+// Unit Test
+
+// Copyright (c) 2020 Digvijay Janartha, Hamirpur, India.
+
+// Parts of Boost.Geometry are redesigned from Geodan's Geographic Library
+// (geolib/GGL), copyright (c) 1995-2010 Geodan, Amsterdam, the Netherlands.
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#include <iostream>
+
+#include <geometry_test_common.hpp>
+
+#include <boost/core/ignore_unused.hpp>
+
+#include <boost/geometry/algorithms/make.hpp>
+
+#include <boost/geometry/algorithms/append.hpp>
+
+#include <boost/geometry/geometries/point.hpp>
+#include <boost/geometry/geometries/point_xy.hpp>
+#include <boost/geometry/geometries/linestring.hpp>
+
+#include <boost/geometry/geometries/concepts/linestring_concept.hpp>
+
+
+#include <boost/geometry/geometries/adapted/c_array.hpp>
+#include <boost/geometry/geometries/adapted/boost_tuple.hpp>
+
+#include <boost/geometry/io/dsv/write.hpp>
+
+#include <test_common/test_point.hpp>
```

Comment:
That is too much of blank lines  :)

Reply:
I thought it was a convention to leave a line if the header is from different directory, like in [segment.cpp](https://github.com/boostorg/geometry/blob/develop/test/geometries/segment.cpp), but I'll fix this :)

Reply:
Not really, let's say it's unintentional legacy and any code should be crafted more carefully, compact, readable. See [#include directives shall be ordered...](https://github.com/boostorg/geometry/wiki/Guidelines-for-Developers#code-structure) guidelines.